modules/transactions.js

```diff
@@ -103,8 +103,9 @@ __private.list = function(filter, cb) {
 	const params = {};
 	const where = [];
 
+	// Removing null characters from data filter to avoid errors in pg-promise library
 	if (filter.data) {
-		params.additionalData = Buffer.from(filter.data, 'utf8');
+		filter.data = Buffer.from(filter.data);
```

Comment:
We don't need `'utf8'` here?

Reply:
By default it's 'utf8' but I add it for clarity.